lib/Gateway.js

```diff
@@ -1321,7 +1321,11 @@ Gateway.prototype.discoverValue = function (node, vId) {
         } else if (cmdClass === CommandClasses.Meter) {
           // https://github.com/zwave-js/node-zwave-js/blob/master/packages/zwave-js/src/lib/commandclass/MeterCC.ts
           // https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/meters.json
-          sensor = Constants.meterType(valueId.ccSpecific.meterType)
+          if (valueId.ccSpecific) {
+            sensor = Constants.meterType(valueId.ccSpecific.meterType)
+          } else {
```

Comment:
Could you add a comment here to explain this? Good catch 👍

Reply:
Added some comments, can you share your thoughts on this?

Reply:
I think it's ok, I would make the same check for multilevel sensor

Reply:
this specific issue is on watt and power metering devices. I expect this would not happen on multilevel. But adding it just in case